addons/core/translations/resources/en-us.yaml

```diff
@@ -436,10 +436,12 @@ credential:
   types:
     username_password: Username & Password
     ssh_private_key: Username & Key Pair
+    json: JSON
     unknown: Unknown
   help:
     username_password: This host requires a username and password to connect.
     ssh_private_key: This host requires a username, public key, and private key to connect.
+    json: This host requires JSON blob to connect.
```

Comment:
I talked with Joshua about what the help text should be and he agreed we should have all Credential type help text match what is in the wireframe. We should go ahead and update all the help text.
<img width="741" alt="Screen Shot 2022-11-18 at 10 47 30 AM" src="https://user-images.githubusercontent.com/29386339/202804774-d85b3df1-5d30-4279-a93b-70df38f3f372.png">